openedx/core/djangoapps/content/course_structures/tasks.py

```diff
@@ -0,0 +1,79 @@
+import json
+import logging
+
+from celery.task import task
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+
+
+log = logging.getLogger(__name__)
```

Comment:
Logging for tasks (running on a worker) should use a different logger than regular LMS code.  If generate_course_structure is not called outside of this context, it should be prefixed with an underscore to make that scope clear.   The worker logging used to use get_task_logger(), but we have now moved to using a fixed name:

log = logging.getLogger('edx.celery.task')